runtime-sdk/src/modules/consensus_accounts/mod.rs

```diff
@@ -84,6 +84,9 @@ pub struct GasCosts {
     pub store_receipt: u64,
     /// Cost of taking a delegation/undelegation receipt.
     pub take_receipt: u64,
+
+    /// Cost of getting delegation info through a subcall.
+    pub delegation: u64,
```

Comment:
You should prepare a PR for setting this in the Sapphire repository.

Reply:
https://github.com/oasisprotocol/sapphire-paratime/pull/573